chainerx/_docs/routines.py

```diff
@@ -1342,6 +1342,27 @@ def _docs_math():
     output array to the input array ``x``.
 
 .. seealso:: :data:`numpy.arctan`
+""")
+
+    _docs.set_doc(
+        chainerx.arctan2,
+        """arctan2(x1, x2)
+Element-wise arc tangent of x1/x2 choosing the quadrant correctly.
```

Comment:
Could you fix to use sphinx math notation?
```
Element-wise arc tangent of :math:`\\frac{x_1}{x_2}` choosing the quadrant
correctly.
```

Reply:
Thank You.